extensions/yaml_support/yaml_support/_parser/base_parser.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from abc import ABCMeta, abstractmethod
+from typing import List, Any
+
+
+class BaseParser(metaclass=ABCMeta):
+    """
+    An abstraction for parsers
+    """
+
+    @abstractmethod
+    def parse(self, code: str) -> List[Any]:
```

Comment:
For example, do we want to define the allowed return type for the parse function?

Reply:
Yes, in the case of the parser it should always return a list of Models, I've replaced Any type with BaseModel which is an abstraction for all Model classes.

Reply:
Can we update for the other 19 instances of Any as well?

Reply:
Hi, I've replaced the Any keyword with exact type hints, I use the Type alias